dtfs-central-api/src/v1/controllers/utilisation-report-service/helpers/keying-sheet-mark-as-helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { FeeRecordEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
+import { NotFoundError } from '../../../../errors';
+import { FeeRecordRepo } from '../../../../repositories/fee-record-repo';
+
+export const getSelectedFeeRecordsAndUtilisationReportForKeyingDataMarkAs = async (
+  reportId: number,
+  selectedFeeRecordIds: number[],
+): Promise<{ selectedFeeRecords: FeeRecordEntity[]; utilisationReport: UtilisationReportEntity }> => {
+  const selectedFeeRecords = await FeeRecordRepo.findByIdAndReportIdWithReport(selectedFeeRecordIds, reportId);
+
+  if (selectedFeeRecords.length !== selectedFeeRecordIds.length) {
+    throw new NotFoundError(`Could not find report with id ${reportId} and with fee records with ids ${selectedFeeRecordIds.join(', ')}`);
```

Comment:
Should this be `Could not find report with id ${reportId} with fee records with ids ...`?